_includes/replacement.html

```diff
@@ -0,0 +1,6 @@
+{%- if site.data.formula[include.item] %}
+{%- assign r_type = "formula" %}
+{%- else %}
+{%- assign r_type = "cask" %}
+{%- endif %}
```

Comment:
You can skip this logic by passing "formula" or "cask" as another parameter, e.g.:
```
{%- include replacement.html item=f.disable_replacement type="formula" %}
```